packages/enzyme-adapter-react-helper/src/getAdapterForReactVersion.js

```diff
@@ -1,30 +1,40 @@
 import semver from 'semver';
 
+function isPreRelease(version) {
+  return semver.valid(version) && semver.SemVer(version).prerelease.length > 0;
+}
+
+function removePreRelease(version) {
+  return semver.inc(version, 'patch');
```

Comment:
This works because a pre-release is **_less_** than the normal release it corresponds to. So, `16.8.0-alpha.1` < `16.8.0`

Reply:
Just realized that there probably should've been a `semver.prerelease` check here (because if it's not a prerelease, the action here doesn't make sense.

Could've been something like:

```js
function makeValidRange(version) {
  return semver.prerelease(reactVersion)
    ? semver.inc(version, 'patch') // Remove prerelease part of the version
    : semver.validRange(reactVersion);
}
```

Reply:
Sounds like a great follow up PR :-D